docs/devices/edge-compute.md

```diff
@@ -0,0 +1,56 @@
+
+## Supported Devices
+
+The following devices have been tested and are known to work as an Edge Compute Device.
+
+|                                                                                        |                                                                                                |
+| -------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------- |
+| **Raspberry Pi 2** ([info][pi-2-info]) ![Raspberry Pi 2][pi-2-image]                   | **Raspberry Pi 3** ([info][pi-3-info]) ![Raspberry Pi 3][pi-3-image]                           |
+| **Advantech UTX-3115** ([info][advantech-info]) ![Advantech UTX-3115][advantech-image] | **BeagleBone Black** ([info][beaglebone-info]) ![BeagleBone Black][beaglebone-image]           |
+| **Intel's FRD Platform** ([info][frd-info]) ![Intel's FRD Platform][frd-image]         | **Dell Edge Gateway 3001** ([info][dell-3001-info]) ![Dell Edge Gateway 3001][dell-3001-image] |
+| **HPE GL10 IoT Gateway** ([info][gl10-info]) ![HPE GL10 IoT Gateway][gl10-image]       |                                                                                                | 
+
+[pi-2-info]: https://www.raspberrypi.org/products/raspberry-pi-2-model-b/
+[pi-2-image]: /images/devices/edge-compute/raspberry-pi-2.png "Raspberry Pi 2"
+[pi-3-info]: https://www.raspberrypi.org/products/raspberry-pi-3-model-b/
+[pi-3-image]: /images/devices/edge-compute/raspberry-pi-3.png "Raspberry Pi 3"
+[advantech-info]: http://www.advantech.com/products/bda911fe-28bc-4171-aed3-67f76f6a12c8/utx-3115/mod_fa00d5cd-7d2b-430b-8983-c232bfb9f315
+[advantech-image]: /images/devices/edge-compute/advantech-utx-3115.png "Advantech UTX-3115"
+[beaglebone-info]: http://beagleboard.org/black
+[beaglebone-image]: /images/devices/edge-compute/beaglebone-black.png "BeagleBone Black"
+[frd-info]: https://www.intel.com/content/www/us/en/internet-of-things/fog-reference-design-overview.html
+[frd-image]: /images/devices/edge-compute/intel-frd.png "Intel's FRD Platform"
+[dell-3001-info]: http://www.dell.com/en-us/work/shop/cty/pdp/spd/dell-edge-gateway-3001/ctoi3001stdus
+[dell-3001-image]: /images/devices/edge-compute/dell-3001.png "Dell Edge Gateway 3001"
+[gl10-info]: https://www.hpe.com/us/en/product-catalog/servers/edgeline-systems/pip.hpe-edgeline-el10-intelligent-gateway.1008670386.html
+[gl10-image]: /images/devices/edge-compute/hpe-gl10.png "HPE GL10 IoT Gateway"
+
+<!--
+Images are 462x322
+-->
+
+## Minimum Requirements
+
+The [Edge Agent](/edge-compute/overview/) must be running on your device and connected to Losant. It runs as a [Docker](https://www.docker.com/what-docker) container and expects some amount of memory to carved out for it. Your mileage may vary, but users have had success with the following specifications. See below for supported devices.
```

Comment:
Add the word be. This should read: expects some amount of memory to be carved out for it.